src/Php81/Resources/stubs/CURLStringFile.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// https://php.watch/versions/8.1/CURLStringFile#CURLStringFile-polyfill
+class CURLStringFile extends CURLFile
+{
```

Comment:
This polyfill is incomplete, because it does not expose the properties. CURLStringFile is a mutable class in PHP.

Reply:
You mean that mutating the `data` property should be taken into account, isn't it? Anything else?

Reply:
Well, we need to expose the 3 public properties, and support mutating them, but also reading them.

Reply:
Since we extend `CURLFile`, which already has `$mime` and `$postname`, they're covered, right?
`$name` would need to be kept in sync with `$data` though.
I suppose this can be done with some magic methods

Reply:
the mime type also impacts the data when using a base64 URL. Also, are they public properties in CURLFile ?

Reply:
> the mime type also impacts the data when using a base64 URL.

Oh, then maybe we can use a generic mime type or no mime type in the data URL, if curl supports that @Ayesh?

> Also, are they public properties in CURLFile ?

Yes they are: https://www.php.net/CURLFile

Reply:
I fear that passing a generic mime type might change how curl sends the data (as CURLFile would then have 2 conflicting values for the mime type)